backend/src/main/java/com/github/JeTSkY1h/security/SecurityConfig.java

```diff
@@ -24,8 +24,8 @@ SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers(HttpMethod.POST, "/api/user", "/api/auth").permitAll()
                 .antMatchers(HttpMethod.GET, "/*", "/index*", "/static/**", "/*.js", "/*.json").permitAll()
-                .antMatchers(HttpMethod.GET, "/api/book/refresh").hasRole("admin")
                 .antMatchers("/**").authenticated()
+                .antMatchers(HttpMethod.GET, "/api/book/refresh").hasRole("admin")
```

Comment:
I think you must put this matcher above the `"/**"` Matcher. Since it was there before(maybe there was some kind of problem), please check with a test, if only admins can refresh